profiles/nikhilbuts.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/36304254?v=4",
```

Comment:
This should be an image of your profile rather than your profile picture. What do you think about this?

![image](https://user-images.githubusercontent.com/51391473/197599390-023ce67a-c274-4449-b303-03359367c625.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/197599390-023ce67a-c274-4449-b303-03359367c625.png",
```

Reply:
Okay @Panquesito7 I will do that.

Thanks!

Reply:
@Panquesito7 I am creating new branch, Here I am getting some conflicts in my folder so.